Fortran/gfortran/regression/DisabledFiles.cmake

```diff
@@ -890,6 +890,9 @@ file(GLOB SKIPPED_FILES CONFIGURE_DEPENDS
   # with procedure designator 'new_t' with explicit interface that cannot be
   # called via an implicit interface
   pr112407a.f90
+
+  # This causes a segmentation fault at compile-time.
```

Comment:
Actually, the segmentation fault happens at run-time.

Reply:
Fixed. Thanks for the clarification.